### ISSUE TITLE
reCAPTCHA fail for second time submit (in version:  1.5.2)

### ISSUE THREAD

**Issue by level1cr3** [michaelvs97/AspNetCore.ReCaptcha#34]
**Describe the bug**
I'm using .net 6 i used previous version everything was fine today(feb 16) i updated the package  and when i'm trying to sumbit 
the form second time i'm getting error. and i'm not able to submit the form. this might be issue on my end as well i'm new to coding
so sorry if it is not in your end

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by level1cr3** [comment]
1. V3 of reCaptcha, latest version of visual studio 2022 and .net 6.
2.  yes(only second time) and this is the error that i can see in google chrome console.
what i mean by second time is . i'm filling the form hitting submit the data will get saved. in db but for second when i fill
the form and hit submit again it won't save the data.

3. no the reCAPTCHA  is not being verified the first time around. for second time it is checking and failing

4
# for first submission it's not checking the recaptcha validation.

In debug: 
modelstate => root key => child node => their is no recaptcha (for first time)

# for second time
In debug:
modelstate => root key => child node => their is recaptcha key. and when i see error message

modelstate is failing second. because recaptcha is not validated.
Your request cannot be completed because you failed Recaptcha verification.

yes modelstate was invalid. no i did not get redicrected i checked error in modelstate by using breakpoint.

**Comment by level1cr3** [comment]
@michaelvs97 

The last thing you asked i don't know how to get it. I'm sorry man i'm new to programming. what exactly i have to get that
if you can tell me step by step

**Comment by level1cr3** [comment]
@michaelvs97 

did configuration from start again. now validation for Recaptcha is being done in the first submission as well so that's good.
But still i don't know why the captcha verification is failing

**Comment by level1cr3** [comment]
> Hey @level1cr3
> 
> Did you manage to resolve this? I'm facing the exact same problem.

Yes I ended up setting interval in js that will generate new token every 30sec. I don't know if this is the correct way though

**Comment by SZLMCL** [comment]
Hi, in my case first time when submit the form the rechaptcha validation is good, but there are another (planned) validation problem with another field. I fix the field and click on submit button second time, in this case the rechaptcha validation fails.
I think every request must have a new token, but during ajax call the token not refreshing and cannot use the old token again.

**Comment by SZLMCL** [comment]
Finally I found the solution, after the first ajax call must call this existing js function: 
updateReCaptcha1()
